qCC/ccGraphicalTransformationTool.cpp

```diff
@@ -137,8 +141,8 @@ void ccGraphicalTransformationTool::advModeToggle(bool state)
 		this->setGeometry(this->x() , this->y(), 0, 0);
 		this->adjustSize(); //adjust size will minimize the display height with the dropdowns not visible
 		this->setGeometry(this->x() + (wPrev - 250), this->y(), 250, this->height());
-		advRotateRefUpdate(0); //index 0 is always the origin
```

Comment:
I found that the state of the combo box was occasionally out of sync with the internal state because the state update was called, but nothing set the control to the new index, everywhere this happened has been replaced with setting the combo box index which will then trigger the same state update function

Reply:
Ah ok. I'm not the one who added this feature, therefore I'll trust your judgement ;)